CHANGELOG.md

```diff
@@ -123,5 +124,23 @@ property should now be used instead for the `EscapeOutput` sniff.
 
 ## [0.3.0] - 2014-12-11
 
+See the comparison for full list.
+
 ### Changed
 - Use semantic version tags for releases.
+
+## [2013-10-06]
+
+See the comparison for full list.
+
+## 2013-06-11
+
+Initial tagged release.
+
+[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.7.0...HEAD
```

Comment:
I don't think that [Keep a CHANGELOG](http://keepachangelog.com/#how-can-i-minimize-the-effort-required-) is really clear on this either way. 

I'm going to go ahead and merge this just like it is so we won't get conflicts when I add the change log for 0.7.1. If you want to discuss the Unreleased section further, go ahead and open up another issue/PR for it.